packages/legend-application-studio/src/components/editor/edit-panel/mapping-editor/PropertyMappingsEditor.tsx

```diff
@@ -180,6 +210,23 @@ export const PropertyMappingsEditor = observer(
         }
       }
     };
+    const setImplementationPropertyMappings = (
+      instanceSetImplementationState.setImplementation as InstanceSetImplementation
```

Comment:
this looks awkward, feel free to modify `InstanceSetImplementationState` to **override** `get setImplementation`